roles/matrix-dynamic-dns/tasks/platform/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- include_tasks: "{{ role_path }}/tasks/platform/centos.yml"
```

Comment:
I suppose there's no good way to run `dhclient` in a Docker container, like we do with the rest of the services?

This would make it more platform-independent and simplify things.

Reply:
AFAIK we actually could use a docker container, it just wouldn't have access to the DHCP operation mode. One of the nice things about ddclient is that it's configurable to run the dynamic dns routine whenever dhcp claims a new IP and I believe this can't be done in a container. It's a little more overhead to run the daemon but not horrible.

I'll make a new PR (and close this one for posterity) with the [container approach](https://hub.docker.com/r/linuxserver/ddclient). I think we should do this anyway because this will interfere least with other services on the host, provide modularity, and be platform agnostic. We can always add the DHCP "mode" back as another option which opts-in to a host-level ddclient.... but I think most people will be happy with the docker solution.

Reply:
Oh BTW I just started a new job and so opensource contributions are not my highest priority right now. This may take me a few weeks to do. I hope to also make PRs against you that enable forward proxying of bridges like mautrix-facebook, enable swap, add a LinkedIn bridge (still in development), and also workaround a docker issue encountered on smaller machines. Just be patient as I get these out!

Reply:
Awesome! Looking forward to these contributions, whenever you get the chance!

Good luck at your new job!